packages/block-library/src/latest-comments/edit.js

```diff
@@ -98,20 +99,26 @@ export default function LatestComments( { attributes, setAttributes } ) {
 					</ToolsPanelItem>
 
 					<ToolsPanelItem
-						hasValue={ () => ! displayExcerpt }
-						label={ __( 'Display excerpt' ) }
+						hasValue={ () => displayContent !== 'excerpt' }
+						label={ __( 'Display Content' ) }
```

Comment:
```suggestion
						label={ __( 'Display content' ) }
```

In this type of UI, Sentence case is generally used instead of Title Case.